plugins/by-name/lazygit/default.nix

```diff
@@ -47,9 +47,8 @@ lib.nixvim.vim-plugin.mkVimPlugin {
       Config file path is evaluated if this value is `1`.
     '';
 
-    config_file_path = defaultNullOpts.mkNullable (
-      with types; either str (listOf str)
-    ) [ ] "Custom config file path or list of custom config file paths.";
+    config_file_path = defaultNullOpts.mkNullable (with types; either str (listOf str)) [
+    ] "Custom config file path or list of custom config file paths.";
```

Comment:
TBH, I'm not a huge fan of either of these formats, but breaking within an empty `[ ]` seems particularly strange. Is that intended @infinisil ?

Reply:
Personally, I'd like to see long multi-arg function calls like this wrapped with one arg per line, e.g.

```nix
config_file_path =
  defaultNullOpts.mkNullable
    (with types; either str (listOf str))
    [ ]
    "Custom config file path or list of custom config file paths.";
```

Reply:
Agreed, please open an issue

Reply:
Sure thing. I've opened https://github.com/NixOS/nixfmt/issues/267 and https://github.com/NixOS/nixfmt/issues/268 for the proposed formatting and weird line-break on empty list, respectively.